phoenix/integration/gitlab.py

```diff
@@ -40,3 +51,40 @@ def get_due_date_issues(days=None):
             if days_remaining == day:
                 notify[open_issue.id] = open_issue
     return notify
+
+
+def parse_report_url(report_url):
+    """Parse report url, return project path and issue ID."""
+    path = urlparse(report_url).path.split('/')
+    if len(path) != 5:
```

Comment:
Not sure how robust is this. Couldn't we use some regex for ensuring the correct format?

Reply:
changed to regex https://github.com/kiwicom/phoenix/pull/10/commits/4f8b795a3eac9e8b446bf15828ac2e84720ba753